### DISCUSSION TITLE
Changed `master` branch to `main`

### DISCUSSION THREAD

**Discussion by jonsterling** [jonsterling/cool-tt#203] [Announcement]
Dear colleagues,

I have changed the `master` branch of the `cooltt` repository to be called `main`. It's high time for this change to role out across our ecosystem, as many have earnestly expressed the offensive character of the term `master` as used in version control. Reality makes words, not the other way around; nonetheless we must acknowledge the way that the words we casually use can reinforce the superstructural aspects of this evil system of imperial domination that we presently live and work within.

To update your local environments, you can run the following commands in your `cooltt` working directory.

```sh
git branch -m master main
git fetch origin
git branch -u origin/main main
git remote set-head origin -a
```

Alternatively you may simply reclone the repository to set up a fresh environment. I hope this causes only minimal inconvenience!

Thanks,
Jon

**Comment by jonsterling** [comment]
Oh, and if you want to change the behavior of `git init` on your machine accordingly, you can run the following command (courtesy of @favonia):

```sh
git config --global init.defaultBranch main
```

**Comment by RobertHarper** [comment]
oh gawd yes, thank you.

(c) Robert Harper.  All Rights Reserved.
